editor/editor_help.cpp

```diff
@@ -2494,11 +2494,32 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 			} else if (link_tag == "annotation") {
 				target_color = link_annotation_color;
 			}
+
 			p_rt->push_color(target_color);
 			p_rt->push_meta("@" + link_tag + " " + link_target);
-			p_rt->add_text(link_target + (link_tag == "method" ? "()" : ""));
-			p_rt->pop();
-			p_rt->pop();
+
+			if (link_tag == "member" && (p_class == "ProjectSettings" || link_target.begins_with("ProjectSettings.") || p_class == "EditorSettings" || link_target.begins_with("EditorSettings."))) {
+				// Special formatting for both ProjectSettings and EditorSettings.
+				String prefix;
+				if (link_target.begins_with("EditorSettings.")) {
+					prefix = "(Editor) ";
+				}
+
+				String setting_name = link_target.trim_prefix("ProjectSettings.").trim_prefix("EditorSettings.");
+				PackedStringArray setting_sections;
+				for (const String &section : setting_name.split("/", false)) {
+					setting_sections.append(EditorPropertyNameProcessor::get_singleton()->process_name(section, EditorPropertyNameProcessor::STYLE_CAPITALIZED));
+				}
```

Comment:
You're opening a can of worms here :)

Two things would need to be taken into account:
- Localization
- The configured capitalization style (`EditorPropertyNameProcessor::get_settings_style()`)

CC @timothyqiu

Reply:
Yeah, using `EditorPropertyNameProcessor::get_settings_style()` would solve both. It takes localization into account.